testsuite/testscripts/test-zeppelin.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import os
+import urllib2
+
+master_url = "http://" + os.environ['ZEPPELIN_MASTER_NODE'] + ":" + os.environ['default_zeppelin_port']
+
```

Comment:
hmmm, I guess we should export a ZEPPELIN_PORT environment variable for the user.